src/pages/authentication/signup/signup.js

```diff
@@ -57,37 +58,36 @@ function SignUp(){
     return(
         <div>
             <Navbar />
-             <form action="">
+             <form  action="" onSubmit={signupHandler}>
```

Comment:
you can remove ```action``` if not being used

Reply:
will work on these changes. thank you nikhil